gammapy/data/observation_stats.py

```diff
@@ -0,0 +1,189 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import astropy.units as u
+from ..stats import Stats
+from gammapy.stats import significance_on_off
+
+__all__ = [
+    'ObservationStats',
+]
+
+
+class ObservationStats(Stats):
+    """Observation statistics.
+
+    Class allowing to summarize observation 
+    (`~gammapy.data.DataStoreObservation) statistics
+
+    Parameters
+    ----------
+    n_on : int
+        number of on events
+    n_off : int
+        number of on events
```

Comment:
There's a bunch of copy & paste errors ... all parameters are described as "number of on events"